app/notifications/utils.py

```diff
@@ -0,0 +1,8 @@
+import requests
+
+
+def confirm_subscription(confirmation_request):
+    url = confirmation_request['SubscribeURL']
+    response = requests.get(url)
+    response.raise_for_status()
+    return confirmation_request['TopicArn']
```

Comment:
Think that assumes it was successful, I was mostly referring to `response` which may raise. We might want to know what error occurred here.